shared/redux/rtk-apis/classworks/classworks.api.ts

```diff
@@ -26,8 +26,20 @@ const classworksApi = projectApi.injectEndpoints({
       },
       invalidatesTags: ["ClassworkResources"],
     }),
+
+    deleteResource: builder.mutation<void, { classroomId: number; resourceId: number }>({
+      query: ({ classroomId, resourceId }) => ({
+        url: `/classworks/${classroomId}/resources/${resourceId}`,
```

Comment:
Let's follow REST principles 


```suggestion
        url: `/classrooms/${classroomId}/resources/${resourceId}`,
```